components/amorphic/HISTORY.md

```diff
@@ -1,3 +1,5 @@
+## 10.4.0
```

Comment:
I think we should briefly mention that `listen(...)` and `startPersistorMode(...)` calls would now require teams to pass logger instead of `sendToLog`  function